backend/api/publishing.py

```
@@ -609,22 +1298,42 @@ async def get_scenario_full(
     scenario_dict = scenario.__dict__.copy()
     scenario_dict['reviews'] = reviews
     scenes = db.query(ScenarioScene).filter(ScenarioScene.scenario_id == scenario_id).order_by(ScenarioScene.scene_order).all()
-    from database.schemas import ScenarioSceneResponse
+    from database.schemas import ScenarioSceneResponse, ScenarioPersonaResponse
```

Ruff (0.14.1)

Redefinition of unused `ScenarioSceneResponse` from line 29

Remove definition: `ScenarioSceneResponse`

(F811)